recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rocksdb" %}
+{% set version = "6.2.2" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha256: 3e7365cb2a35982e95e5e5dd0b3352dc78573193dafca02788572318c38483fb
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
```

Comment:
Always helpful to leave a comment here about the reason behind this decision. I would put:
 https://abi-laboratory.pro/index.php?view=timeline&l=rocksdb as a comment.

Reply:
I updated the recipe to include your comment.